src/util/mod.rs

```diff
@@ -165,3 +171,8 @@ impl From<secp256k1::Error> for Error {
     }
 }
 
+impl From<psbt::Error> for Error {
```

Comment:
Please add a `#[doc(hidden)]` attribute ([C-HIDDEN][1]).

[1]: https://rust-lang-nursery.github.io/api-guidelines/documentation.html#c-hidden

Reply:
Should that be added to all the `From<*> for Error` `impl`s?

Reply:
Ideally

Reply:
@jeandudey let's open up another PR after this gets merged to fix all of them?

Reply:
Yes, it's up for another PR.

Reply:
Going to wait until this one's merged to do this for all `From`s at once

Reply:
Maybe we should change `SimpleDecoder` to always use `util::Error` rather than being parameterized, and add a `pub use util::Error` at the top-level, in the same PR.

Reply:
https://github.com/rust-bitcoin/rust-bitcoin/pull/127 added `#[doc(hidden)]` everywhere - we should do so in this PR then as well.